include/bluetooth/gatt_dm.h

```diff
@@ -292,6 +292,11 @@ int bt_gatt_dm_start(struct bt_conn *conn,
  * This function continues service discovery.
  * Call it after the previous data was released by @ref bt_gatt_dm_data_release.
  *
+ * When the service UUID had been set when service discovery was started using
+ * @ref bt_gatt_dm_start. This function discover the next instance of the same service
+ * if there are more instances on the GATT server. You can use this function to discover
+ * all services instances with given UUID.
```

Comment:
Maybe something like this:
The service UIDD is set when you start service discovery using the ... function. If there are more instances of the same service on the GATT server, it discovers the next instance. Use this function to discover all service instances with the given UIDD.

Reply:
Ok, I changed it